README.md

```diff
@@ -90,6 +90,39 @@ Runtime Environment (node v10.15.1 on linux)
 CLI version 3.0.1
 ```
 
+## Install locally via yarn (alternative install)
+
+required: current version of yarn installed (should also work with npm)
+
+create a file `package.json`within your project with the following content
```

Comment:
There's a blank missing after the second backtick (`). Consider a capital "C" at the beginning, and maybe a colon at the end?

```suggestion
Create a file `package.json` within your project with the following content:
```